profile/src/main/kotlin/com/imashnake/animite/profile/dev/internal/Constants.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("MaxLineLength")
```

Comment:
Maybe just suppress this for the one line that is too long?

Reply:
Both are, and likely more to come.